Cargo.toml

```diff
@@ -18,6 +18,9 @@ build = "build.rs"
 edition = "2018"
 publish = false
 
+[workspace]
+members = ["clippy_dev", "clippy_lints"]
```

Comment:
Can we consider ``clippy_dev`` as a member of the workspace? neither ``clippy`` nor ``clippy_lints`` depend on it.

Reply:
Agreed I'll remove it. Actually I can just remove this line cause path dependencies are automatically included.

There's another issue I (or someone) need to figure out here. Conflicting target output between tests.

Reply:
The test failures are definitely weird. I guess they are caused by the macro aux crate being compiled as a workspace member and then again for every test.